ibc-primitives/src/utils/macros.rs

```diff
@@ -1,51 +0,0 @@
-//! Macros used across the ibc-rs crates.
```

Comment:
Nice clean-ups!

Reply:
Before this wasn't possible as `let PATTERN = value else { ... }` is stabilized in 1.65 :slightly_smiling_face: